README.md

```diff
@@ -15,19 +16,12 @@ Role Variables
 Required variables:
 
 - `openstack_volume_size`: Size of the volume in GB
-- `openstack_volume_vmname`: If defined, the name of the instance the volume should be attached to
 - `openstack_volume_name`: The name of the volume
-- `openstack_volume_device`: The device name of the volume in the instance (default: automatically determined by Openstack)
-- `openstack_volume_type`: The type of volume
-- `openstack_volume_source`: The name of the volume to clone.
-
-If `openstack_volume_source` is specified, the role will create an intermediate
-volume snapshot, named after `openstack_volume_name` and suffixed with
-`-source`, and use this snapshot as the source of the new volume.
-
-When `openstack_volume_vmname` is passed, the snapshot used for the volume
-creation is cleaned up (if it exists) before attaching the volume to the
-instance.
+- `openstack_volume_device`: The device name of the volume in the instance (default: automatically determined by OpenStack)
+- `openstack_volume_type`: The type of volume (default: automatically determined by OpenStack)
+- `openstack_volume_source`: If defined, the name of an existing volume to use as the source of the volume creation. An temporary volume snapshot will be created, named after `openstack_volume_name` and suffixed with `-source` and cleaned after the volume is created.
```

Comment:
```suggestion
- `openstack_volume_source`: If defined, the name of an existing volume to use as the source of the volume creation. A temporary volume snapshot will be created, named after `openstack_volume_name` and suffixed with `-source` and cleaned after the volume is created.
```

Reply:
It's a bit unclear why the user needs to know that it will be created if it's then deleted.

Reply:
I agree, this is internal implementation detail and has no user-facing value except in the scenario where `openstack_volume_wait` is False as this role will leave an uncleaned snapshot. [e96b3e2](https://github.com/ome/ansible-role-openstack-volume-storage/pull/5/commits/e96b3e2d4b1b432e82cabf4eb372d94becd3b343) should address this